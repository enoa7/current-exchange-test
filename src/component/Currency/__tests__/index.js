import React from "react";
import { create } from "react-test-renderer";
import CurrencyComponent from "../index";

describe("Currency component", () => {
	test("it matches the snapshot", () => {
		const mockup = {
			data: {
				name: "name",
				value: "value",
			},
		};
		const component = create(
			<CurrencyComponent data={mockup} currencyValue={1.23} onDeleteSymbol={() => console.log("delete")} />
		);
		expect(component.toJSON()).toMatchSnapshot();
	});
});
