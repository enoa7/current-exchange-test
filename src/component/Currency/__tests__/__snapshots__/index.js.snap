// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Currency component it matches the snapshot 1`] = `
<div
  className="currency-item"
>
  <div
    className="currency-wrapper"
  >
    <div>
      <div
        className="currency-detail"
      >
        <div
          className="currency-detail-code"
        />
        <div
          className="currency-detail-value"
        >
          NaN
        </div>
      </div>
      <div
        className="currency-name"
      >
        undefined - undefined
      </div>
      <div
        className="currency-rate"
      >
        1 undefined = undefined
      </div>
    </div>
    <div
      className="pointer-click"
      onClick={[Function]}
    >
      (-)
    </div>
  </div>
</div>
`;
